### DISCUSSION TITLE
azimuthal data (image)

### DISCUSSION THREAD

**Discussion by valex470** [kinverarity1/lasio#590] [Ideas]
Hello!
Do you have any plans to add support a visualization of azimuthal data (images) in lasio?
Thanks!

**Comment by valex470** [comment]
Find some decision in this repo: https://github.com/andymcdgeo/Petrophysics-Python-Series

Actually in here
https://github.com/andymcdgeo/Petrophysics-Python-Series/blob/b6bcfb17f026e8b612cf64ed24d408bb69290d4e/13%20-%20Displaying%20LWD%20Image%20Data.ipynb

**Comment by kinverarity1** [comment]
Thanks for the suggestion but no, lasio is mainly about reading data in. Things like visualisation itself are best handled in other libraries or scripts - the example you linked is brilliant!
